Resources/Prototypes/_White/Loadouts/Jobs/engineering.yml

```
@@ -0,0 +1,55 @@
+  id: ClothingUniformJumpsuitEngineerPolar
+  category: JobsEngineeringAAUncategorized
+  cost: 0
+  exclusive: true
+  requirements:
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutEngineeringUniforms
+    - !type:CharacterJobRequirement
+      jobs:
+        - StationEngineer
+  items:
+    - ClothingUniformJumpsuitEngineerPolar
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)